object/slicer/options.go

```diff
@@ -56,7 +56,8 @@ func (x *Options) SetCopiesNumber(copiesNumber uint32) {
 }
 
 // SetPayloadBuffer sets pre-allocated payloadBuffer to be used to object uploading.
-// For better performance payloadBuffer length should be MaxObjectSize from NeoFS.
+// The payloadBuffer should have length at least MaxObjectSize+1 from NeoFS,
```

Comment:
I don't think it's correct, we have explicit `SetObjectPayloadLimit`.

Reply:
fixed